apps/web/src/pages/farms/index.tsx

```diff
@@ -37,12 +38,8 @@ const FarmsPage = () => {
   const { address: account } = useAccount()
   const { chosenFarmsMemoized } = useContext(FarmsV3Context)
   const cakePrice = usePriceCakeUSD()
-  const { data: farmV2 } = useFarmsV2Data()
+  const { regularCakePerBlock: cakePerBlock, totalRegularAllocPoint } = useFarms()
 
-  const { totalRegularAllocPoint, cakePerBlock } = farmV2 || {
```

Comment:
@chef-jojo small problem here need your advice on to fix. so @Woody810 s old custom hook "useFarmsDataV2", was used specifically to fetch the "totalRegularAlloc" and "CakePerBlock" as this data was not available in the useFamrs() Hook. i removed this redundant hook like you asked, and tried to get these values from the normal useFarms hoook. however for whatever reason, although these params are exported from the redux farmSelector state, they return as undefined from the useFarms hook. 

so you can see here that instead of using cakePerBlock i use regularCakePerBlock as its defined in this hook. the reason ive done this is because from loggin it seems that this value was identical to cakePerBlock only it was in ether form instead of wei, so in the farmsPerSec calc below i just format it correctly and it gives the same result for the calcs, as before my changes.

so the only issue i need to resolve is how to get the totalRegularAllocPoint data in the V2 farms hook. its strange because this data is available in the aptos useFarmsHook (although that one doesnt seem to use redux state). so maybe you could help me on this. I feel maybe this is why @Woody810 created that extra custom hook, to get this data but this is just me speculating.

Reply:
As @ChefBingbong  mentioned, it was not possible to retrieve `totalRegularAlloc` and `CakePerBlock` from the useFarms() V2 Hook. That's why I created an additional function called `useFarmsDataV2`. Thank you for Bingbong's assistance.